test/unit/custom_spec.js

```diff
@@ -0,0 +1,99 @@
+import { buildGetDocumentParams } from './test_utils';
```

Comment:
This file is missing the required license header, see e.g. https://github.com/mozilla/pdf.js/blob/master/src/license_header.js.